cardano-testnet/src/Testnet/Start/Cardano.hs

```diff
@@ -244,6 +263,8 @@ cardanoTestnet
 
   -- Make Non P2P topology files
   forM_ (zip [1..] portNumbers) $ \(i, myPortNumber) -> do
+    -- TODO: if the user provided its own configuration file, and requested a P2P topology file,
```

Comment:
Can you turn this into an issue so we can track it?

Reply:
Done here: https://github.com/IntersectMBO/cardano-node/issues/6150